docs/integrations/databases/opentelemetry/redis-opentelemetry.md

```diff
@@ -171,3 +198,37 @@ Use this dashboard to:
 - Review the RDBMemory Usage, events, RDB, and AOF events.
 
 <img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/Redis-Logs.png' alt="Redis Logs dashboards" />
+
+### Overview
+
+The **Redis - Overview** dashboard provides an at-a-glance view of the Redis server status and error logs, along with database metrics.
+
+Use this dashboard to review keyspace hit, connected clients, Uptime, Evicted vs Expired Keys, and Commands Per Seconds.
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/Redis-Overview.png' alt="Redis Overview dashboards" />
+
+### Cluster Operations
+
+The **Redis - Cluster Operations** Operations dashboard provides a high-level view of the activity and health of Master and Slaves on your network. Dashboard panels display visual graphs and detailed information on cache rate and most importantly, resource usage.
+
+Use this dashboard to review keyspace hit, Cache miss rate by host, CPU - System vs User usage by host, and Network Usage - Inbound/Outbound by Host.
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/Redis-Cluster-Operations.png' alt="Redis Cluster Operations dashboards" />
+
+### Cluster Overview
+
+The **Redis - Cluster Overview** dashboard provides a high-level view of cluster infrastructure, comparisons, and trends.
+
+Use this dashboard to review Average Master Uptime, Connected Slaves, Redis Uptime by Host, Commands by Host, and Instantaneous OPS per sec by Host.
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/Redis-Cluster-Overview.png' alt="Redis Cluster Overview dashboards" />
+
+### Command Stats Delta
+
+The **Redis - Command Stats Delta** dashboard provides an at-a-glance view of commands running on your Redis server.
+
+Use this dashboard to review String Commands, Key Commands, Hash Commands, Sets Commands, Sorted Sets Commands, List Commands, Transaction Commands, and Scripting Commands.
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Redis-OpenTelemetry/Redis-Command-Stats-Delta.png' alt="Redis Command Stats Delta dashboards" />
```

Comment:
This image is not rendering in my local build. Can you verify that the URL is correct?